src/main/java/frc/robot/commands/EjectCoralCommand.java

```diff
@@ -0,0 +1,43 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.ArmSubsystem;
+import frc.robot.subsystems.GamePieceManipulatorSubsystem;
+
+/**
+ * Command to eject the coral from the robot.
+ */
+public class EjectCoralCommand extends Command {
+  private final GamePieceManipulatorSubsystem gamePieceManipulatorSubsystem;
+  private final ArmSubsystem armSubsystem;
+
+  /**
+   * Constructs a new EjectCoralCommand
+   * 
+   * @param manipulatorSubsytem manipulator subsystem
+   * @param armSubsystem arm subsystem
+   */
+  public EjectCoralCommand(GamePieceManipulatorSubsystem manipulatorSubsytem, ArmSubsystem armSubsystem) {
+    this.gamePieceManipulatorSubsystem = manipulatorSubsytem;
+    this.armSubsystem = armSubsystem;
+    addRequirements(gamePieceManipulatorSubsystem, armSubsystem);
+  }
+
+  @Override
+  public void execute() {
+    gamePieceManipulatorSubsystem.ejectCoral();
+
+  }
+
+  @Override
+  public boolean isFinished() {
+    return !gamePieceManipulatorSubsystem.isCoralInPickupPosition();
```

Comment:
I think this condition would end the command too early. We probably just have to let the drive choose when to stop since we have no sensor to know when this action is complete.